tests/unit/command/test_plots.py

```diff
@@ -240,6 +240,36 @@ def test_plots_path_is_quoted_and_resolved_properly(
     assert expected_url in out
 
 
+def test_should_pass_template_dir(tmp_dir, dvc, mocker, capsys):
```

Comment:
Both of these tests are not useful. I'll suggest testing actual behaviour instead.

Reply:
> Both of these tests are not useful.

@skshetry Could you elaborate on why are not useful? 

> I'll suggest testing actual behaviour instead if you can or get rid of them.

Behavior is tested in https://github.com/iterative/dvc-render/blob/main/tests/test_templates.py#L39